ui/lib/apps/Statement/utils/useSchemaColumns.ts

```diff
@@ -1,28 +1,29 @@
 import { useEffect, useState } from 'react'
+import { useRequest } from 'ahooks'
 
 import client from '@lib/client'
-import { cache } from '@lib/utils/callableCache'
 
 const statementsTableColumnsGet = client
   .getInstance()
   .statementsTableColumnsGet.bind(client.getInstance())
 
 export const useSchemaColumns = () => {
-  const [isLoading, setLoading] = useState(true)
   const [schemaColumns, setSchemaColumns] = useState<string[]>([])
+  const { data: resp, loading } = useRequest(statementsTableColumnsGet, {
```

Comment:
I believe `useRequest` and similar hooks (like useSWR) are not well adapted if they are introduced directly. For example, as you can see, we must create a binding manually `statementsTableColumnsGet` to use, which is unpleasant. This cache is also not shared cross modules. The adoption of such libraries needs some more refinements :)

Reply:
I'm open to implementations that even use `Proxy` to transparently handle these things, if it is needed.

Reply:
That's right! Let me think about it.